data_steward/utils/bq.py

```diff
@@ -0,0 +1,17 @@
+from google.cloud import bigquery
+
+
+def get_client(project_id=None):
+    """
+    Get a client for a specified project.
+    """
+    if project_id is None:
+        return bigquery.Client()
```

Comment:
I dislike letting the user "forget" to set the project id.  This goes back to CH's concern about overusing environment variables.  I'd rather require a project_id be specified here and raise an error if it isn't.  Then the user has to provide this parameter and can't accidentally use the wrong dataset because they didn't check what the project environment variable is set to.  A developer can grab it from an environment variable if needed, but I'd like to curb this habit.

Reply:
right now I left it because all the notebooks use it, and in notebooks, it only gets the project_id from by using the service account key.  When we update the notebooks we can make this method much stricter.

Reply:
It'll be on my agenda.  They don't do it because we let them.

Reply:
I'll compromise, how about adding a logging statement nagging them to start specifying the project_id before creating the client?
```
if project_id:
    return bigquery.Client(project=project_id)
else:
    log.(naggging statement)
    return bigquery.Client()
```

Reply:
sure, I can add that.